src/deepsparse/image_classification/README.md

```diff
@@ -0,0 +1,274 @@
+# Image Classification Inference Pipelines
+
+
+[DeepSparse] Image Classification integration allows serving and benchmarking 
+sparsified image classification models. This integration allows for leveraging 
```

Comment:
```suggestion
sparsified image classification models. This integration enables leveraging 
```
nit: avoiding repeating the word "allows".

Reply:
Good catch! thanks